apiary.apib

```diff
@@ -3566,6 +3566,12 @@ Get a list of work orders.
 + Request (application/json)
 
     + Attributes (object)
+        + filter (object, optional)
+            + state (enum[string], optional)
```

Comment:
This should be array of state, right? To allow multiple states at once?

Reply:
Let's go for the multiple states. 👍